src/sentry/utils/silo/common.py

```diff
@@ -25,11 +26,30 @@ def find_source_paths():
                     yield os.path.join(dirpath, filename)
 
     def find_class_declarations():
+        """Find the target class declarations.
+
+        For each module-class pair in `target_names`, check the Python source file
+        corresponding to that module. If the named class is declared in that module,
+        yield it.
+        """
+        targets = defaultdict(set)
+        for (module_name, class_name) in target_names:
+            targets[class_name].add(module_name)
```

Comment:
We have two endpoint classes named `JiraIssueUpdatedWebhook` in different modules. This fixes a bug where the script was ignoring one arbitrarily.